MergeSharpWebAPI/RetryPolicy.cs

```diff
@@ -0,0 +1,10 @@
+using Microsoft.AspNetCore.SignalR.Client;
+
+public class RetryPolicy : IRetryPolicy
+{
+    private const int ReconnectionWaitSeconds = 5;
+    public TimeSpan? NextRetryDelay(RetryContext retryContext)
+    {
+        return TimeSpan.FromSeconds(ReconnectionWaitSeconds);
+    }
+}
```

Comment:
I'm assuming this class is just used to set how often the backend tries to reconnect to the server??

Reply:
Yea, its a class for the retry policy which determines how often and what behavior needs to be done when retrying to establish a connection. In this case you you keep retrying to establish a connection every 5 seconds indefinitely